Header.c

```diff
@@ -81,7 +122,9 @@ void Header_writeBackToSettings(const Header* this) {
    }
 }
 
-MeterModeId Header_addMeterByName(Header* this, const char* name, int column) {
+MeterModeId Header_addMeterByName(Header* this, const char* name, unsigned int column) {
+   assert(column < HeaderLayout_getColumns(this->headerLayout));
+
    Vector* meters = this->columns[column];
 
    char* paren = strchr(name, '(');
```

Comment:
Shouldn't it be `meter->mode = mode;` in the `mode` assignment below?

Reply:
It was somehow correct, but the returned mode was nowhere used; refactored the function